yt_idv/scene_annotations/block_histogram.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+import traitlets
+from OpenGL import GL
+
+from yt_idv.scene_annotations.base_annotation import SceneAnnotation
+from yt_idv.scene_data.block_collection import BlockCollection
+
+
+class BlockHistogram(SceneAnnotation):
+    """
+    A class that computes and displays a histogram of block data.
+    """
+
+    name = "block_histogram"
+    data = traitlets.Instance(BlockCollection)
+    bins = traitlets.CInt(64)
+    min_val = traitlets.CFloat(0.0)
+    max_val = traitlets.CFloat(1.0)
```

Comment:
More of a note for the future -- the data attribute here has a `data.min_val` and `data.max_val`, which are the raw min/max across all blocks. Since the compute shader will operate on normalized data, makes sense to keep these min_val, max_val values at 0,1. but when drawing the result (however that happens), but when it comes time to draw the result, if there are labels for the binning axis, it'd be nice to be able to re-scale those to the actual data range (which will be easy since `data.min_val` and `data.max_val` exist already).

Reply:
Oh, awesome!  Thank you.  I forgot that the data was already normalized.

Reply:
So in principle we could do some fancy work to make these in the normalized coordinates too.